packages/fontaine/test/index.spec.ts

```
@@ -1,301 +1,81 @@
-    )).toMatchInlineSnapshot(`
-      [
-        {
-          "family": "Arial",
-          "index": 0,
-          "properties": {},
-        },
-      ]
-    `)
-  })
-})
+import { describe, it, expect, beforeEach } from 'vitest';
```

ESLint

[error] Expected "expect" to come before "it".

(perfectionist/sort-named-imports)